policy/policy_manager_test.go

```diff
@@ -52,13 +52,6 @@ func Test_Payloadmanager_init_success2(t *testing.T) {
 	}
 }
 
-func Test_Payloadmanager_dup_policy_name(t *testing.T) {
-
-	if _, err := Initialize("./test/pfduptest/", make(map[string]string), nil, true); err == nil {
```

Comment:
It's ok that you removed this test but you should replace it with a test of the new function.

Reply:
Will do.